stylesheet.css

```diff
@@ -0,0 +1,210 @@
+body {
+    margin: 10px;
+    padding: 0;
+    background-color: whitesmoke;
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: white;
+}
+
+.homeBar {
+    width: 100%;
+    overflow: auto;
+
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.searchLink {
+    display: flex;
+    width: 65%;
+    align-items: center;
+    padding-left: 10%;
+    flex-wrap: wrap;
+}
+
+.searchBar {
+    width: 75%;
+    height: 25px;
+}
+
+.icons {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    height: 25px;
+    padding-right: 10px;
+}
+ 
+.fab {
+    color: red;
+    font-size: 35px;
+}
+
+.fas {
+    padding: 0 5px 0 5px;
+    font-size: 20px;
+    height: 25px;
+}
+
+.container {
+    display: flex;
+    display: -webkit-flex;
+    justify-content: space-around;
+    margin: 0 2px 0 2px;
+    ;
+}
+
+.right-container {
+    float: right;
+    align-items: center;
+    width: 35%;
+    overflow: 0;
+}
+
+.left-container {
+    width: 70%;
+    float: left;
+    margin-right: 20px;
+}
+
+.userData {
+    background-color: white;
+    width: 100%;
+}
+
+#subscribe {
+    background-color: red;
+    color: white;
+}
+
+.channel {
+    display: flex;
+    flex-direction: none;
+}
+
+.channel1 {
+    float: left;
+}
+
+.channel2 {
+    float: left;
+}
+
```

Comment:
- [ ] You could group those two selectors together since they have the same property